docs/mobile/sdk-components-reference.mdx

```diff
@@ -95,7 +95,7 @@ namespace App1.Components
 
 #### Component Dependencies
 
-Components are instantiated by [dependency injection](sdk-dependency-injection.mdx) when the layout is rendered. This means that a component can also list dependencies that have been registered with autofac, such as the [`ILayoutModel<MapView>`](sdk-component-service-interactions.mdx) in it's constructor, and have them be passed at runtime. This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will either need to provide that dependency somehow.
+Components are instantiated by [dependency injection](sdk-dependency-injection.mdx) when the layout is rendered. This means that a component can also list dependencies that have been registered with autofac, such as the [`ILayoutModel<MapView>`](sdk-component-service-interactions.mdx) in its constructor, and have them be passed at runtime. This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will either need to provide that dependency somehow.
```

Comment:
This also means that if a class associated with a component has application dependencies, such as a reference to the `MapOperations` for map interactions, then the component will **either** need to provide that dependency somehow.

Use of 'either' implies that an 'or' will follow... Maybe you don't need the word 'either'?

Reply:
As with the first comment, I was only interested in fixing the spelling here.